### PR TITLE
Fixed 26 containers

### PR DESCRIPTION
**Pull request by SvitlanaYaroslavicka1**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue
Fixed containers
## Testing approach

Fixed 26 containers
## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by andrii-kuzmichov — COMMENTED** [review]
Revert changes to package-lock.json

File: src/containers/event-draft.js [hunk 1]
```diff
@@ -142,18 +144,50 @@ const mapStateToProps = state => ({
...
     edit_event: (data, onError, onSuccess) =>
-      dispatch(edit_event(data, onError, onSuccess)),
+      dispatch(editEvent(data, onError, onSuccess)),
     delete: (eventId, reason) =>
-      dispatch(change_event_status(eventId, reason, eventStatusEnum.Deleted)),
-    publish: data => dispatch(publish_event(data)),
-    get_categories: () => dispatch(get_categories()),
+      dispatch(changeEventStatus(eventId, reason, eventStatusEnum.Deleted)),
+    publish: data => dispatch(publishEvent(data)),
+    get_categories: () => dispatch(getCategories()),
```
Comment (andrii-kuzmichov) [inline src/containers/event-draft.js:151]: let rename and add details to naming

File: src/containers/event-edit-wrapper.js [hunk 2]
```diff
@@ -50,11 +51,30 @@ const mapStateToProps = state => ({
...
 
 const mapDispatchToProps = dispatch => ({
-  get_event: id => dispatch(get_event(id)),
+  get_event: id => dispatch(getEvent(id)),
   approveUser: (userId, eventId, buttonAction) =>
     dispatch(approveUser(userId, eventId, buttonAction)),
-  get_Categories: () => dispatch(get_categories()),
+  get_Categories: () => dispatch(getCategories()),
   reset: () => dispatch(resetEvent()),
 });
```
Comment (andrii-kuzmichov) [inline src/containers/event-edit-wrapper.js:59]: the same as above

File: src/containers/event-filter.js [hunk 3]
```diff
@@ -29,14 +30,14 @@ class EventFilterWrapper extends Component {
...
   };
 
-  onSubmit = filters => {
-    filters = filterHelper.trimUndefinedKeys(filters);
+  onSubmit = initialFilters => {
+    const filters = filterHelper.trimUndefinedKeys(initialFilters);
     const filterCopy = { ...this.props.events.filter };
     Object.entries(filters).forEach(function ([key, value]) {
       switch (key) {
         case "page":
```
Comment (andrii-kuzmichov) [inline src/containers/event-filter.js:38]: create constants for it and next values

File: src/containers/event-filter.js [hunk 4]
```diff
@@ -117,11 +116,30 @@ const mapStateToProps = state => ({
 
 const mapDispatchToProps = dispatch => {
   return {
-    get_categories: () => dispatch(get_categories()),
+    get_categories: () => dispatch(getCategories()),
     reset_events: () => dispatch(reset("event-filter-form")),
```
Comment (andrii-kuzmichov) [inline src/containers/event-filter.js:120]: rename

File: src/containers/event-item-view.js [hunk 5]
```diff
@@ -72,23 +73,55 @@ const mapStateToProps = state => ({
...
+    dispatch(changeEventStatus(eventId, reason, eventStatusEnum.Deleted)),
   get_users_inventories_by_event_id: eventId =>
-    dispatch(get_users_inventories_by_event_id(eventId)),
+    dispatch(getUsersInventoriesByEventId(eventId)),
   get_inventories_by_event_id: eventId =>
-    dispatch(get_inventories_by_event_id(eventId)),
-  get_unitsOfMeasuring: () => dispatch(get_unitsOfMeasuring()),
+    dispatch(getInventoriesByEventId(eventId)),
+  get_unitsOfMeasuring: () => dispatch(getUnitsOfMeasuring()),
   reset: () => dispatch(resetEvent()),
```
Comment (andrii-kuzmichov) [inline src/containers/event-item-view.js:90]: rename, fixing formatting and add details

File: src/containers/event-list.js [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
 import { withRouter } from "react-router";
 import EventList from "../components/event/EventList/Event-list";
 import SpinnerWrapper from "./spinner";
 import {
-  get_events,
-  reset_events,
-  updateEventsFilters,
+  getEvents,
+  // resetEvents,
+  // updateEventsFilters,
```
Comment (andrii-kuzmichov) [inline src/containers/event-list.js:12]: Do you need it?

File: src/containers/event-list.js [hunk 7]
```diff
@@ -76,10 +79,24 @@ const mapStateToProps = state => {
 
 const mapDispatchToProps = dispatch => {
   return {
-    get_events: filter => dispatch(get_events(filter)),
+    get_events: filter => dispatch(getEvents(filter)),
```
Comment (andrii-kuzmichov) [inline src/containers/event-list.js:82]: rename

File: src/containers/event-validate-form.js [hunk 8]
```diff
@@ -1,27 +1,27 @@
-export const validateEventForm = values => {
-  if (!values) return values;
+const validateEventForm = values => {
+  const valuesCopy = values;
+  if (!values) return valuesCopy;
 
   if (!values.isPublic) {
```
Comment (andrii-kuzmichov) [inline src/containers/event-validate-form.js:5]: use destructuring for this file

**Review by andrii-kuzmichov — COMMENTED** [review]
Fix all snake case variables, try to install and reuse lodash(isEmpty, isNill, isEqual)

File: src/containers/event-list.js (hunk 6 above)
Comment (SvitlanaYaroslavicka1) [reply src/containers/event-list.js:12]: No

File: src/containers/delete-comment.js [hunk 9]
```diff
@@ -9,14 +10,16 @@ import AddComment from "./add-comment";
...
+    super(props);
+    this.state = { info: true };
+  }
 
   changeInfo = () => {
-    this.setState({
-      info: !this.state.info,
-    });
+    this.setState(previousState => ({
+      ...previousState,
```
[the remark this answers is not in the document]
Comment (SvitlanaYaroslavicka1) [reply src/containers/delete-comment.js:20]: Of course

**Pull request closed without merging**